### DISCUSSION TITLE
Shortcut features and parsing

### DISCUSSION THREAD

**Discussion by Ambushfall** [johnlindquist/kit#1516] [Ideas]
There isn't a nice table showcasing all of the modifiers that are shortcut-able.

I reckon creating all options would be a good starter.

Also, I'd love to suggest just straight out allowing VK_KEY Parsing from the frontmatter.

**Comment by johnlindquist** [comment]
VK_KEY parsing?

We're limited by these options:

https://www.electronjs.org/docs/latest/api/accelerator

The shortcut parser converts them into that format. I think I'll expose an "Accelerator" metadata that just sends the raw string for people who don't want to fuss with the "friendly" Shortcut metadata
